tests/vic-machine.bats

```diff
@@ -123,16 +137,21 @@ teardown() {
     run "$docker" "${docker_cmd[@]}" start "$name"
     assert_success
 
+
     run govc vm.destroy "$name"
     assert_success
 
     run govc datastore.rm -f "$name"
     assert_success
 
+    # if we're here we should grab the logs cause we're gonna wipe out the VCH
+    get_logs "$installer_output"
+
     # test reinstall with same name
     params+=(-force -key ./${vch_name}-key.pem -cert ./${vch_name}-cert.pem)
 
     run "$installer" "${params[@]}"
+    installer_output="$output"
```

Comment:
Saves off the installer output so that the correct logs are grabbed when teardown is called. Notice that it happens before assert_success so that the logs will be grabbed regardless of failure/success of the test. Without it, the first installation's logs would be grabbed by teardown instead. I do realize that it does not appear straightforward, as I'm using global state to pass information so the flow of the test is not terribly obvious.. wish there was a better way but given the constraints of testing with Bash I don't think there is